gcal_helpers/helpers.py

```diff
@@ -687,6 +703,8 @@ def generate_rss(cal_dict):
 
     time_now = get_time_now()
 
+    cal_dict['items'] = sort_by_date(filter_duplicate_guids(cal_dict['items']))
```

Comment:
Do you mind adding a comment just above this describing what/why this is being done?